kratos/processes/calculate_embedded_nodal_variable_from_skin_process.h

```diff
@@ -599,12 +599,16 @@ class CalculateEmbeddedNodalVariableFromSkinProcess : public Process
 
         // Populate the modelpart with all the nodes in NodesMap
         // Note that a temporary vector is created from the set to add all nodes at once
-        PointerVectorSet<Node> tmp;
+        PointerVectorSet<
```

Comment:
why the defaults do not work here?

Reply:
Nodes constructor with `key` is not available (it throws an error), if someone does not specify the second template argument to the `PointerVectorSet`, then it uses `SetIdentityFunction`, which will need to create a copy of the node by using the `key` and node consturctor which throws an error. This is only problematic for `Node` type only.

Reply:
This is the ctor being called in this case.

https://github.com/KratosMultiphysics/Kratos/blob/8fa8e65ef65a737e19652c1d7db2933f1f6efb69/kratos/includes/node.h#L117-L128